app/models/team_performance.rb

```diff
@@ -0,0 +1,72 @@
+class TeamPerformance
+# Memo: calculates Team's Performance Score for Supervisor's Dashboard
+
+  def initialize(team)
+    @team = team
+  end
+
+  def score
+    @score = 0
+
+    if @team.comments.any?
+      comments_score
+    elsif Time.now-2.days > Season.current.starts_at && Time.now < Season.current.ends_at
+      # Maybe we should change the +3 score temporarily for the upcoming week,
+      # because before Wed 16th comments-without-text were not being saved.
+      @score += 3
+    else
+      @score += 0
+    end
+
+    if @team.activities.any?
+      activity_score
+    elsif Time.now-2.days > Season.current.starts_at && Time.now < Season.current.ends_at
+      @score += 3
+    else
+      @score += 0
+    end
+    evaluate_performance
+  end
+
+  def comments_score
+    latest_comment = @team.comments.ordered.first
+    if latest_comment.created_at <= Time.now-5.days
```

Comment:
I tried that initially, but as I remember, I got unexpected behaviour regarding the UTC. I searched the code base to see how that was handled elsewhere. I didn't find the `days.ago` format, so I choose to go the safe way.